apps/web/src/components/Alert.tsx

```diff
@@ -3,12 +3,25 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { ReactNode } from 'react';
 
 type AlertProps = {
+  color?: 'blue' | 'yellow' | 'red';
   children: ReactNode;
 };
 
-export const Alert = ({ children }: AlertProps) => {
+export const Alert = ({ color, children }: AlertProps) => {
+  let containerClasses = '';
+  switch (color) {
+    case 'red':
+      containerClasses = 'bg-bcRedError py-4  text-left flex items-center rounded';
+      break;
+    case 'yellow':
+      containerClasses = 'bg-bcYellowCream py-4  text-left flex items-center rounded';
+      break;
+    default:
+      containerClasses = 'bg-bcLightBlueBackground py-4  text-left flex items-center rounded';
```

Comment:
can probably just make `py-4 text-left flex items-center rounded` a base class since it doesn't change and have the colours be a key value object